UPGRADE.md

```diff
@@ -1,3 +1,77 @@
+### UPGRADE FROM 1.0.0-RC.5 TO 1.0.0-RC.6
+
+1. The `Version20201208105207.php` migration was added which extends existing adjustments with additional details (context). Depending on the type of adjustment, additionally defined information are:
+ * Taxation details (percentage and relation to tax rate)
+ * Shipping details (shipping relation)
+ * Taxation for shipping (combined details of percentage and shipping relation)
+
+ This data is fetched based on two assumptions:
+ * Order level taxes relates to shipping only (default Sylius behaviour)
+ * Tax rate name has not change since the time, the first order has been placed
+
+ If these are not true, please adjust migration accordingly to your need. To exclude following migration from execution run following code: 
+    ```
+    bin/console doctrine:migrations:version 'Sylius\RefundPlugin\Migrations\Version20201208105207' --add
+    ```
+
+1. Add traits that enhance Adjustment and Shipment models from Sylius. These traits are not covered by 
+the backward compatibility promise and it will be removed after update Sylius to 1.9. It is a duplication of a logic 
+from Sylius to provide proper adjustments handling.
+
+```php
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\RefundPlugin\Application\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Sylius\Component\Order\Model\Adjustment as BaseAdjustment;
+use Sylius\RefundPlugin\Entity\AdjustmentInterface;
+use Sylius\RefundPlugin\Entity\AdjustmentTrait;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="sylius_adjustment")
+ */
+class Adjustment extends BaseAdjustment implements AdjustmentInterface
+{
+    use AdjustmentTrait;
+}
+```
+
+```php
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\RefundPlugin\Application\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\Common\Collections\ArrayCollection;
+use Sylius\Component\Core\Model\Shipment as BaseShipment;
+use Sylius\RefundPlugin\Entity\ShipmentInterface;
+use Sylius\RefundPlugin\Entity\ShipmentTrait;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="sylius_shipment")
+ */
+class Shipment extends BaseShipment implements ShipmentInterface
+{
+    use ShipmentTrait;
+
+    public function __construct()
+    {
+        parent::__construct();
+
+        /** @var ArrayCollection<array-key, BaseAdjustmentInterface> $this->adjustments */
```

Comment:
Probably a minor thing, but `BaseAdjustmentInterface` does not exist in the namespaces